webrtc/videoroom/lib/sfu/media_event.ex

```diff
@@ -0,0 +1,201 @@
+defmodule Membrane.SFU.MediaEvent do
+  def create_peer_accepted_event(peer_id, peers) do
+    peers =
+      Enum.map(peers, fn peer ->
+        %{id: peer.id, metadata: peer.metadata, midToTrackMetadata: peer.mid_to_track_metadata}
+      end)
+
+    %{type: "peerAccepted", data: %{id: peer_id, peersInRoom: peers}}
+    |> do_create(peer_id)
+  end
+
+  def create_peer_denied_event(peer_id) do
+    %{type: "peerDenied"}
+    |> do_create(peer_id)
+  end
+
+  def create_peer_joined_event(peer_id, metadata, mid_to_track_metadata) do
+    %{
+      type: "peerJoined",
+      data: %{
+        peer: %{
+          id: peer_id,
+          metadata: metadata,
+          midToTrackMetadata: mid_to_track_metadata
+        }
+      }
+    }
+    |> do_create(:broadcast)
+  end
+
+  def create_peer_left_event(peer_id) do
+    %{
+      type: "peerLeft",
+      data: %{
+        peerId: peer_id
+      }
+    }
+    |> do_create(:broadcast)
+  end
+
+  def create_signal_event(peer_id, {:signal, {:candidate, candidate, sdp_m_line_index}}) do
+    %{
+      type: "candidate",
+      data: %{
+        candidate: candidate,
+        sdpMLineIndex: sdp_m_line_index,
+        sdpMid: nil,
+        usernameFragment: nil
+      }
+    }
+    |> do_create(peer_id)
+  end
+
+  def create_signal_event(peer_id, {:signal, {:sdp_offer, offer}}) do
+    %{
+      type: "sdpOffer",
+      data: %{
+        type: "offer",
+        sdp: offer
+      }
+    }
+    |> do_create(peer_id)
+  end
+
+  def create_error_event(to, msg) do
+    %{
+      type: "error",
+      data: %{
+        message: msg
+      }
+    }
+    |> do_create(to)
+  end
+
+  defp do_create(event, to) do
+    event
+    |> serialize()
+    |> then(fn event -> {:media_event, to, event} end)
```

Comment:
`Room` expects from first element of returned tuple to be `:sfu_media_event` instead of `:media event`